cli.md

```diff
@@ -6,6 +6,12 @@ permalink: /cli/
 order: 50
 ---
 
+The GAPID package contains a command line tool named `gapit` that  can be used
```

Comment:
Shouldn't this be autogenerated somehow? How are you going to update this doc in the future?

Reply:
I didn't touch the generated part. To re-generate we can simply prepend the auto generated portion with the header. This can be done manually when updating this file, or if we setup an automated way to do that, we could make the header an input to it. Feel free to make changes to the header, however, since this is visible now, I figured it is best to put something in place now, rather than wait for some automatic way.